app/helpers/editions_helper.rb

```diff
@@ -36,6 +36,11 @@ def browse_options_for_select(grouped_collections)
     end
   end
 
+  def format_conversion_select_options(edition)
+    possible_target_formats = Edition.convertible_formats - [edition.artefact.kind]
+    possible_target_formats.map{|format_name| [format_name.humanize, format_name]}
```

Comment:
Its a bit late now, but our styleguide says we should have spaces before and after the curl braces.